ui/__config__/webpack.config.js

```diff
@@ -65,7 +65,20 @@ const config = {
             react: 'preact',
         },
     },
-    resolveLoader: { modules: [path.resolve(ui, '__tools__'), 'node_modules'] },
+    resolveLoader: {
+        modules: [
+            path.resolve('node_modules', '@guardian', 'guui', 'lib', 'loaders'),
+            path.resolve(
+                '..',
+                'node_modules',
+                '@guardian',
+                'guui',
+                'lib',
+                'loaders'
+            ),
```

Comment:
Infuriating repetition just in case `@guardian/guui` is installed in the root `node_modules` as opposed to the `ui` workspace.